.github/release-drafter.yml

```
@@ -23,6 +23,7 @@ categories:
       - "dependency"
       - "docs"
 
+category-template: "$TITLE: "
```

```suggestion
category-template: "$TITLE:"
```

@ssbarnea that's a bug. Thanks for noticing.